vizro-core/examples/kpi/assets/css/custom.css

```diff
@@ -0,0 +1,56 @@
+#page-header {
+  padding-left: 4px;
+}
+
+.kpi-card-ref.card {
+  padding: 8px;
+}
+
+.kpi-card-ref h4 {
+  margin: 0;
+}
+
+.kpi-card-ref > span {
+  display: flex;
+  font-weight: 600;
+  gap: 4px;
+}
+
+.kpi-card-ref .delta-pos {
+  color: #1a85ff;
```

Comment:
Are these colours meant to stay the same in dark and light theme? I guess yes?

Reply:
Yes, I purposely chose a colour palette that works well on both themes :)